gcp/modules/gcp-stackdriver-monitoring/resources/alert_policies/backup_exporter_errors.json

```diff
@@ -0,0 +1,38 @@
+{
+  "display_name": "Error Check for Backup-exporter",
+  "combiner": "OR",
+  "conditions": [
+    {
+      "condition_threshold": {
+        "aggregations": [
+          {
+            "alignment_period": {
+              "seconds": 60,
+              "nanos" : 0
+            },
+            "cross_series_reducer": "REDUCE_MAX",
+            "group_by_fields": [
+              "metric.label.log"
+            ],
+            "per_series_aligner": "ALIGN_MEAN"
+          }
+        ],
+        "comparison": "COMPARISON_GT",
+        "duration": {
+          "seconds": 0,
```

Comment:
This might be very flappy - basically, it will fire for every error, no? Maybe we should consider longer value?